rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+    @person.add_rental(self)
+    @book.add_rental(self)
```

Comment:
- [ ] This code `@person.add_rental(...)` does not add the rental itself to the person. Rather it is interacting with the `@book` and `@person` instance variables. Kindly change to something like this instead:

  ```
  person.rentals << self
  book.rentals << self
  ```
  This way we would be adding the rental itself to the actual book and person objects.

Reply:
Yeaaap!!! Thanks bro @ShadyShawkat  i get it!